app/docs/0.10.x/configuration.md

```diff
@@ -12,6 +12,7 @@ title: Configuration Reference
 - [Custom Nginx configuration & embedding Kong](#custom-nginx-configuration-embedding-kong)
   - [Custom Nginx configuration](#custom-nginx-configuration)
   - [Embedding Kong](#embedding-kong)
+  - [Serving API and Website](#serving-api-and-website)
```

Comment:
I suggested this super-short sub-title to @salazar  - I now see that it is a little too short :)